src/pages/Order/Ordered.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Ordered.scss';
+
+function ordered() {
+  return (
+    <div className="ordered">
+      <div className="orderedTop">
+        <div className="orderedLogo">
+          <img src="images/ordered/logo(252525).png" />
+        </div>
+        <h2>
+          고객님의 주문이 완료되었습니다.
+          <br />
+          감사합니다.
+        </h2>
+      </div>
+      <div className="orderedBottom">
+        <table>
+          <thead>
+            <tr>
+              <th>주문번호</th>
+              <th>상품명</th>
+              <th>수량</th>
+              <th>배송지</th>
+              <th>수령희망일</th>
```

Comment:
```suggestion
  {ORDER_INFO_LIST.map(title => {
                return <th key={title}>{title}</th>;
              })}
```
이 부분은 변하지 않는 데이터이니 상수 데이터로 구현해주세요~